resources/base/crds/sts.min.io_policybindings.yaml

```diff
@@ -4,7 +4,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.15.0
-    operator.min.io/version: v6.0.2
+    operator.min.io/version: v5.0.15
```

Comment:
Is this correct? Shouldn't it be `v6.0.3`?

Reply:
yes, it is intentonal

Reply:
that's right, good catch @aqeelat!, Fixed on https://github.com/minio/operator/pull/2304